meta-refkit-extra/doc/robotics-uarm-metal.rst

```diff
@@ -0,0 +1,293 @@
+Introduction
+============
+
+This is a detailed walkthrough on how to configure MoveIt! framework to
+control an uArm Metal mechanical arm. All the needed config files
+are provided by this layer.
+
+Prerequisites
+=============
+
+Your uArm Metal's controller must be programmed to support a small subset
+of G-codes sent via its serial. Particularly it needs to understand the codes:
+
+M203
+   Attach all servos.
+
+M204
+   Detach all servos.
+
+G205 <base_angle> <left_angle> <right_angle> <hand_angle>
+   Set position for all four servos. The code accepts four parameters which
+   are PWM signal widths for the respective servos in microseconds.
+
+P200
+   Return current servo positions as raw data from the servos' analog pins.
+   The command should reply with a line like::
+
+     RET <base_analog_angle> <left_analog_angle> <right_analog_angle> <hand_analog_angle>
+
+As soon as the controller is ready to receive a new G-code it must signal that
+by sending `ok` to serial.
+
+Setting up
+==========
+
+1. Create an image containing the packages `uarmmeta-moveit-config`,
+   `python3-pyserial` and `python3-asyncio`, e.g. by putting this line
+   in your `local.conf`::
+
+     IMAGE_INSTALL_append = " uarmmeta-moveit-config python3-pyserial python3-asyncio"
+
+   and running::
+
+     $ bitbake refkit-image-common
+
+2. Plug the mechanical arm in a USB port.
```

Comment:
I know it may sound obvious but can you add one step to indicate that the image should be installed on the target platform (and if we have a standard installation guide, you could point to it).